app/models/course.ts

```diff
@@ -44,6 +44,7 @@ export default class CourseModel extends Model {
   @attr('string') declare shortName: string;
   @attr('string') declare slug: string;
   @attr('string') declare testerRepositoryFullName: string;
+  @attr('string') declare visibility: 'public' | 'private';
```

Comment:
Let's add accessors like `visibilityIsPrivate` and `visibilityIsPublic` and use them in the other places where we're accessing this. Keeps things consistent with `releaseStatusIs...` and prevents any mistakes in using the enum value. 

Code like `rejectBy('visibility', 'prvate')` won't cause a type error but `rejectBy('visibilityIsPrvate')` will